dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task04330WorkflowTaskAddLanguageIdColumn.java

```diff
@@ -0,0 +1,275 @@
+
+
+package com.dotmarketing.startup.runonce;
+
+import static com.dotcms.util.CollectionsUtils.*;
+
+import com.dotcms.util.CollectionsUtils;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotcms.util.CollectionsUtils'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)